python/ray/air/execution/_internal/actor_manager.py

```diff
@@ -269,7 +269,7 @@ def _actor_task_failed(
         """
         tracked_actor = tracked_actor_task._tracked_actor
 
-        if isinstance(exception, RayActorError):
+        if isinstance(exception, (RayActorError, ActorUnavailableError)):
```

Comment:
Don't need to change air code?

This is a way to show that existing code can continue working

Reply:
ok, reverted this. but we still need the new `ray.kill()`

Reply:
I guess this is a bug they current have? Prefer to separate it to a different PR.